docs/repositories-configure/codacy-configuration-file.md

```diff
@@ -72,6 +72,15 @@ To ignore files, you must use the [Java glob syntax](https://docs.oracle.com/jav
 | `**.resource`      | All `.resource` files across all your repository             |
 | `**/*.resource`    | All `.resource` files in all directories and sub-directories |
 
+For example:
```

Comment:
I'm not 100% sure we need an example here. Thoughts?

Reply:
I like having an example here. 👍 
Do you think we should be more descriptive? Or maybe it's too much? 🤔 

```suggestion
For example, to ignore the file `test/README.md` and all `.resource` files in all directories and sub-directories add the following to your Codacy configuration file:
```

Reply:
I think we can avoid redundancy here. There should be enough context in the table above.